dump1090.c

```diff
@@ -667,14 +667,14 @@ void showHelp(void) {
 "| dump1090 ModeS Receiver     %45s |\n"
 "-----------------------------------------------------------------------------\n"
 "--device-index <index>   Select RTL device (default: 0)\n"
-"--gain <db>              Set gain (default: max gain. Use -10 for auto-gain)\n"
```

Comment:
I am very confused. -10 is what the help says and what the init file uses and what piaware-support configures, but -100 is indeed what's in the code. Going to have to look at this one a bit more.